fakeredis.py

```diff
@@ -980,6 +980,10 @@ def delete(self, *names):
                 continue
         return deleted
 
+    @_locked
```

Comment:
`self.delete` already takes the lock, so it's unnecessary for this function to be locked.